testbed/macos/Runner.xcodeproj/project.pbxproj

```diff
@@ -366,7 +378,7 @@
 			);
 			runOnlyForDeploymentPostprocessing = 0;
 			shellPath = /bin/sh;
-			shellScript = "\"$FDE_ROOT\"/tools/build_flutter_assets \"$PROJECT_DIR\"/..\n";
+			shellScript = "\"$FLUTTER_ROOT\"/packages/flutter_tools/bin/macos_build_flutter_assets.sh";
```

Comment:
Don't change this yet for testbed; that would break the documented (and heavily used by me) local engine workflow.

Later we can make the Flutter scripts support local engine and migrate the workflow and docs to that.